development/generate-attributions/override-yarnrc.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const targetPath = '.yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs';
+const lines = fs.readFileSync(process.argv[2], 'utf8').split('\n');
+let inPluginsSection = false;
+let inTargetPluginBlock = false;
+const result = [];
+lines.forEach((line) => {
+  if (line.trim() === 'plugins:') {
+    inPluginsSection = true; // Start of plugins section
```

Comment:
Nit: Do you think these comments are useful? It would be better to make the code as self-describing as possible rather than explain what each line of code is doing.